bundles/org.openhab.automation.jsscripting/src/main/resources/node_modules/@jsscripting-globals.js

```diff
@@ -1,189 +1,189 @@
 // ThreadsafeTimers is injected into the JS runtime
 
 (function (global) {
-    'use strict';
```

Comment:
Is indentation changed from 2 to 4 spaces on purpose?

Reply:
Not really. I have applied the semistandard JavaScript linter style (that is the same linter we use at the openHAB JavaScript library).
The changes are mostly intendation, double/single quotes as well as let and const instead of var (before linting, var and let/const were used mixed).

Reply:
I did this to have the same codestyle, and I personally prefer the "clean" code style that linters enforce.

Reply:
Not needed, was just curious if done by purpose to achieve some specific code style in JavaScript, or by accident. Thanks for explaining.